spec/liquid/liquid_renderer_spec.rb

```diff
@@ -163,13 +163,13 @@
       it 'has the fields from one plugin form' do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name']
+        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name', 'country']
       end
 
       it "checks with the member's liquid data" do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com'
+        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com', country: 'US'
```

Comment:
After doing some research here, what was happening had nothing to do with the form. `outstanding_fields` is built off the required values of a form, which include `country` by default now. So, including the country on the member seems to be the best fix, since the behavior of the `outstanding_fields` method appears to be correct.